src/main/java/pojo/Browser.java

```diff
@@ -1,5 +1,30 @@
 package pojo;
 
-public class Browser {
 
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Browser {
+    public static WebDriver openBrowser(String url, String browser){
+   WebDriverManager.chromedriver().setup();
+        WebDriver driver = null;
+        if(browser.equals("chrome"))
+        {
+            //System.setProperty("webdriver.chrome.driver","E:\\automation-selenium\\chrome-headless-shell-win64\\chrome-headless-shell.exe");
```

Comment:
Please uncomment the code or remove if the god is not required

Reply:
Changes Done...